app/Http/Controllers/OrganizationController.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Bde\Organization;
+use Illuminate\Http\Request;
+
+class OrganizationController extends Controller
+{
+    public function index(){
+
+        $search = request()->query('search');
+
+        $organization = Organization::where('name', 'like', "%{$search}%")
+            ->orWhere('short_name', 'like', "%{$search}%")
+            ->get();
+
+        $associations = $organization->where('association', '=', 1); // 1 = association
+
+        if ($associations->isEmpty()) {
+            $associations_tab = [];
+        }else{
+            $associations_tab = $associations->map(function ($asso) {
+                return [
+                    'id' => $asso->id,
+                    'short_name' => $asso->short_name ? $asso->short_name : $asso->name,
+                    'name' => $asso->name,
+                    'logo_url' => $asso->getLogoPath()
+                ];
+            })->values();
+        }
+
+
+        $clubs = $organization->where('association', '=', 0); // 0 = club
+
+        if ($clubs->isEmpty()) {
+            $clubs_tab = [];
+        }else{
+            $clubs_tab = $clubs->map(function ($club) {
+                return [
+                    'id' => $club->id,
+                    'short_name' => $club->short_name ? $club->short_name : $club->name,
+                    'name' => $club->name,
+                    'logo_url' => $club->getLogoPath()
+                ];
+            })->values();
+        }
+
+
+        return response()->json(['data' => [
+            'associations' => $associations_tab,
+            'clubs' => $clubs_tab,
+        ]])->setEncodingOptions(JSON_PRETTY_PRINT);
+    }
+
+    public function show($id){
+        $per_page = request()->query('per_page');
+
+        $organization = Organization::all()->where('id', '=', $id)->first();
+
+        if ($organization == null) {
+            return response()->json(['data' => []])->setEncodingOptions(JSON_PRETTY_PRINT);
+        }
+
+        $organization_tab = [
+            'short_name' => $organization->short_name,
+            'name' => $organization->name,
+            'description' => $organization->description,
+            'website_link' => $organization->website_link,
+            'facebook_link' => $organization->facebook_link,
+            'twitter_link' => $organization->twitter_link,
+            'instagram_link' => $organization->instagram_link,
+            'discord_link' => $organization->discord_link,
+            'email' => $organization->email,
+            'logo_url' => $organization->getLogoPath(),
+        ];
```

Comment:
Tu peu convertir plus vite en tableau avec une méthode (https://laravel.com/docs/11.x/eloquent-serialization#serializing-to-arrays)